packages/compiler/src/server/tspconfig/completion.ts

```diff
@@ -204,10 +363,107 @@ export async function provideTspconfigCompletionItems(
               return item;
             });
             result.push(...enums);
+          } else if (cur.type === "string") {
+            // extends
+            if (relativeFiles && relativeFiles.length > 0) {
+              for (const file of relativeFiles) {
+                if (target.siblings.includes(file)) {
+                  continue;
+                }
+
+                const item: CompletionItem = {
+                  label: file,
+                  kind: CompletionItemKind.Field,
+                  textEdit: getNewTextAndPosition(file, source, tspConfigPosition),
+                };
+                result.push(item);
+              }
+            }
           }
         }
       });
 
     return distinctArray(result, (t) => t.label);
   }
 }
+
+/**
+ * Get the new text value
+ * @param sourceQuoteType input quote type(single, double, none)
+ * @param formatText format text value
+ * @returns
+ */
+function getNewTextValue(sourceQuoteType: string, formatText: string): string {
+  let newText: string = "";
+  if (sourceQuoteType === "QUOTE_SINGLE") {
+    newText = `${formatText}'`;
+  } else if (sourceQuoteType === "QUOTE_DOUBLE") {
+    newText = `${formatText}"`;
+  } else {
+    newText = `"${formatText}"`;
+  }
+  return newText;
+}
+
+/**
+ * Get the position of the new text
+ * @param newText  the new text
+ * @param source source text
+ * @param tspConfigPosition original position
+ * @returns
+ */
+function getNewTextAndPosition(
+  newText: string,
+  source: string,
+  tspConfigPosition: Position,
+): TextEdit {
+  return TextEdit.replace(
+    Range.create(
+      Position.create(tspConfigPosition.line, tspConfigPosition.character - source.length),
+      Position.create(tspConfigPosition.line, tspConfigPosition.character + newText.length),
+    ),
+    newText,
+  );
+}
+
+/**
+ * Find a set of relative paths to files with the same suffix
+ * @param rootPath The root path of the current configuration file
+ * @param fileExtension  File extension
+ * @param configFile Configuration file name
+ * @param tspConfigFile Full path of configuration file (including file name)
+ * @returns
+ */
+function findFilesWithSameExtension(
```

Comment:
I think we just need to show the folder and files with desired ext in cur folder. no need to do recursive. we dont want to do too much IO for intellisense